app/components/schools/participants/coc_status_table_row.rb

```diff
@@ -37,6 +37,10 @@ def date_column_value
         end
       end
 
+      def path_ids
```

Comment:
Here's a method I added so we're not relying on Rails already knowing the path to build links, it's more explicit and easier to test.